.github/workflows/add-missing-labels/get-added-component-packages.js

```diff
@@ -0,0 +1,67 @@
+const { execSync } = require('child_process')
+
+const scriptPrefix = 'check-added-packages:';
+
+/**
+ * Gets the file content from a specific Git reference
+ * @param {string} ref git ref
+ * @param {string} filePath path to the desired file
+ * @returns string with the file content
+ */
+async function getPackageJsonAtRef (ref, filePath) {
+    try {
+        const fileContent = execSync(`git show ${ref}:${filePath}`).toString();
+
+        return JSON.parse(fileContent);
+    } catch (err) {
+        throw new Error(`${scriptPrefix} Failed to fetch package.json from ${ref}. "${err.message}"`)
+    }
+}
+
+/**
+ * Compares the entries of two objects and returns the added keys
+ * @param {object} baseObj dependencies at base brach
+ * @param {object} currentObj dependencies at current branch
+ * @returns Array with the names of added dependencies
+ */
+function getNewDependencies (baseObj = {}, currentObj = {}) {
```

Comment:
When we upgrade to Node 22 I think we can use a native Set method for this:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/difference

No changes required for now 😄

Reply:
Didn't knew that one! Pretty useful!!!

Reply:
Added a todo comment